mappings/636e8ee4539b5eb2e39f97e820443b9dfa9dec87c736debec19202bf504d44.json

```diff
@@ -0,0 +1,54 @@
+{
+    "subject": "636e8ee4539b5eb2e39f97e820443b9dfa9dec87c736debec19202bf504d44",
+    "url": {
+        "sequenceNumber": 0,
+        "value": "https://www.pizzamandans.com/",
```

Comment:
Hi @fredjl3,

please put policyId of the token to the website so we can verify the project.

Thanks!